evalml/model_understanding/prediction_explanations/_user_interface.py

```diff
@@ -125,6 +125,7 @@ def _make_text_table(
     original_features,
     top_k,
     include_shap_values=False,
+    include_expected_value=False,
```

Comment:
I think we can get rid of this parameter since it's not used.

Reply:
not sure how that got past....